docs/content/getting-started/protocol.en.md

```diff
@@ -0,0 +1,88 @@
+
+## Asynchronous Transfers
+
+When a Beneficiary VASP receives Travel Rule transfer information from the Originating VASP through the TRISA protocol it may not always be able to respond immediately with any needed corrections or fill in for the missing/incorrect Beneficiary information provided. Thus the Beneficiary VASP should be able to respond back, at a later time, with the correct beneficiary information or provide errors if appropriate e.g. the beneficiary address is not owned by the Beneficiary VASP.
+
+The Beneficiary VASP will return a [control flow response](https://trisa.dev/data-payloads/#a-pending-message) to the transfer message to notify the Originating VASP that they will handle the compliance transfer in an asynchronous manner. The reply is also composed as a secure message with digitally signed timestamps for auditing purposes. Inside of the transaction payload, the Beneficiary VASP will provide additional information about when the Originating VASP can expect a response. The Originating VASP should delay any further action or communication until either the expected time frame expires or they receive a follow-on response from the Beneficiary VASP.
+
+```proto
+message Pending {
+    string envelope_id = 1;       // TRISA envelope ID
+    string received_by = 2;       // recipient or recipient VASP name
+    string received_at = 3;       // when request was received (RFC3339)
+    string message = 4;           // optional message for counterparty
+    string reply_not_after = 5;   // when response will be returned (RFC3339)
+    string reply_not_before = 6;  // response will not be sent before (RFC3339)
+    string extra_json = 7;        // extra data (JSON-formatted)
+    Transaction transaction = 15; // original transaction for reference
+}
+```
+
+The Originating VASP should be prepared to receive such asynchronous callbacks from the Beneficiary VASP and take appropriate measures to handle it by caching the original message and awaiting a subsequent transfer message from the Beneficiary VASP. All messages that refer to the same blockchain transaction will contain the same envelope ID so that all messages may be linked together. Messages are ordered by the digitally signed timestamps in the payload, so it is important for integrators to ensure they are correctly decrypting payloads and sending correctly modified responses with new timestamps.
+
+Once the Beneficiary VASP has completed their compliance processing, they should initiate a TRISA transfer back to the originating VASP that accepts the original transfer by providing the original or corrected compliance information, or rejects it by providing an error in the secure envelope. This will give the Originating VASP conclusive information to take follow on actions like executing the transaction on the chain or handling the error and following the appropriate workflow.
```

Comment:
```suggestion
Once the Beneficiary VASP has completed their compliance processing, they should initiate a TRISA transfer back to the originating VASP that accepts the original transfer by providing the original or corrected compliance information or rejects it by providing an error in the secure envelope. This will give the Originating VASP conclusive information to take follow on actions like executing the transaction on the chain or handling the error and following the appropriate workflow.
```